packages/backend-core/src/tenancy/context.js

```diff
@@ -1,6 +1,25 @@
 const env = require("../environment")
 const { Headers } = require("../../constants")
 const cls = require("./FunctionContext")
+const { getCouch } = require("../db")
+const { getDeployedAppID, getDevelopmentAppID } = require("../db/conversions")
+const { isEqual } = require("lodash")
+
+// some test cases call functions directly, need to
+// store an app ID to pretend there is a context
+let TEST_APP_ID = null
+
+const ContextKeys = {
+  TENANT_ID: "tenantId",
+  APP_ID: "appId",
+  // whatever the request app DB was
+  CURRENT_DB: "currentDb",
+  // get the prod app DB from the request
+  PROD_DB: "prodDb",
+  // get the dev app DB from the request
+  DEV_DB: "devDb",
+  DB_OPTS: "dbOpts",
```

Comment:
Doesn't feel quite right for this app stuff to live in `tenancy/context.js`. Because this stuff has the potential to grow over time (user context, licensing context etc), it might make sense to refactor this a bit now to set in place a more modular structure e.g. 
```
src/context
- index.js // the context keys, shared functions
- app.js
- tenancy.js
```

Reply:
Thought about this, might just change the directory from `tenancy` to `context` as tenancy should live under the context directory anyway.